static/js/tools/choropleth/choropleth.tsx

```diff
@@ -0,0 +1,351 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Creates and manages choropleth rendering.
+ */
+
+import React, { Component } from "react";
+import axios from "axios";
+import * as d3 from "d3";
+
+class ChoroplethMap extends Component {
+    constructor(props: any) {
+        super(props);
+
+        // Redirect to basic url if none provided.
+        if (window.location.search === "") {
+            window.location.search = "statVar=Count_Person_Employed&perCapita=t&geoId=country/USA";
+        }
+        
+        // Add default state and bind function contexts.
+        const urlParams = new URLSearchParams(window.location.search);
+        this.state = {
+            geojson: [],
+            perCapita: (urlParams.has("perCapita")
+                && (["t","true", "1"]).includes(
```

Comment:
Let's just stick with the presence of "perCapita" param as true -- better yet, simplify to "pc". We're in control of the app and can control which option is used.

Reply:
It doesn't always make sense to only consider perCapita values though. I will change the param to pc